front/app/containers/IdeasShow/components/RightColumnDesktop/index.tsx

```diff
@@ -95,25 +93,26 @@ const RightColumnDesktop = ({
           background={colors.background}
           mb="12px"
         >
-          {showGreyBox && (
-            <>
-              <StyledReactionControl
-                styleType="shadow"
-                ideaId={ideaId}
-                size="4"
-              />
-              <Box pb="23px" mb="23px" borderBottom="solid 1px #ccc">
-                {participationContext &&
-                  ideaIsInParticipationContext &&
-                  votingConfig?.getIdeaPageVoteInput({
-                    ideaId,
-                    participationContext,
-                    compact: false,
-                  })}
-              </Box>
-              {commentingEnabled && <Buttons />}
-            </>
-          )}
+          {ideaIsInParticipationContext ||
+            (commentingEnabled && (
+              <>
```

Comment:
If an idea-based phase has commenting disabled but reacting (or voting) enabled, this section would get incorrectly hidden right? :thinking: Or maybe I'm misunderstanding - the parentheses confuses me a bit here.